Lombiq.NodeJs.Extensions/scripts/markdown-to-msbuild.js

```diff
@@ -0,0 +1,129 @@
+﻿const findRecursively = require('./find-recursively');
+const fs = require('fs');
+const fsPromises = require('node:fs/promises');
+const markdownlint = require('markdownlint');
+const path = require('path');
+const process = require('process');
+const textlint = require("textlint");
+
+const markdownlintConfig = {
+    default: true,
+    MD013: false,
+    MD033: {
+        allowed_elements: [
+            // A special element in GitHub to indicate a keyboard key. Other Markdown formatters that don't support it will
+            // safely ignore the tags and render the content as inline text without adverse effects.
+            'kbd',
+        ]
+    }
+};
+const textLintConfig = {
+    exclude: [
+        // License files are full of legalese, which can't and shouldn't be analysed with tools made for normal prose.
+        'License.md',
+        // The wwwwroot directory contains built and vendor assets. Any Markdown file there is not our responsibility.
+        'wwwroot'
+    ],
+    rules: [
+        "common-misspellings",
+        "doubled-spaces",
+        //"no-dead-link", // Disabled becuase it can't ignore relative links and can't reliably verify GitHub URLs.
```

Comment:
Oh, no! Becuase is back!

Reply:
I guess I have no choice but to conquer the world and make it the official spelling by decree...

Reply:
You shouldn't change the spelling just for it to adhere to your personal preference becuase that would be a very US English thing to do.

Reply:
![image](https://user-images.githubusercontent.com/4007293/180334629-3bd6ce37-9f9d-4ca0-b6a8-d08104789e97.png)

Reply:
:D

Reply:
Maybe we should have that spell-checking plugin after all, @DAud-IcI ? 😉

Reply:
Not needed, the `common-misspellings` rule already detects "becuase". But textlint is only running on _md_ files so it won't do anything with JS and C# comments.

Reply:
Good news, I have installed a text replace tool ([espanso](https://espanso.org/); originally I planned to use AutoHotKey which I had in the past, but I figured let's install one I can use both on Windows and Linux and so far it looks good enough) so now if I write because, because or because it will be corrected into "because" seamlessly as I type.

Reply:
I'll miss it. Adding code suggestions for fixing "becuase" always gave me a sense of purpose.

Reply:
Don't worry, espanso has no effect on copy-paste and I have saved this little nest-egg for future PRs with you. 😁 
[spare-becuase-stockpile-for-zoltan.txt](https://github.com/Lombiq/NodeJs-Extensions/files/9172356/spare-becuase-stockpile-for-zoltan.txt)

Reply:
:D